### PR TITLE
looker studio connection with cloud.

### PR DESCRIPTION
**Pull request by incfly**
Signed-off-by: Jianfei Hu <hujianfei258@gmail.com>

### CODE REVIEW

**Review by justindeguzman — APPROVED** [review]

**Comment by robot-ch-test-poll4** [comment]
This is an automated comment for commit 12d198662f77ffa7dce4589657975b9febf1264a with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/1437/12d198662f77ffa7dce4589657975b9febf1264a/ci_running.html)
The overall status of the commit is 🟢 success

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>Docs Check</td><td>Builds and tests the documentation</td><td>🟢 success</td></tr>
<tr><td>Vercel</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>🟢 success</td></tr>
</table>

File: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md [hunk 1]
```diff
@@ -121,3 +121,13 @@ Finally, you should be able to see the introspected table structure and adjust t
 <br/>
 
 Now you can proceed with exploring your data or creating a new report! 
+
+## ClickcHouse Cloud Connection
+
+When using ClickHouse Cloud, you must first file a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL protocol.
```
Comment (slvrtrn) [inline docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md:127]: Shall we avoid "must" in this case? And protocol -> interface (https://clickhouse.com/docs/en/interfaces/mysql)

```suggestion
When using ClickHouse Cloud, create a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL interface.
```

File: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md [hunk 2]
```diff
@@ -121,3 +121,13 @@ Finally, you should be able to see the introspected table structure and adjust t
 <br/>
 
 Now you can proceed with exploring your data or creating a new report! 
+
+## ClickcHouse Cloud Connection
```
Comment (slvrtrn) [inline docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md:125]:
```suggestion
## Using Looker Studio with ClickHouse Cloud
```

I'd also move this right below "On-premise ClickHouse server setup"

File: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md [hunk 3]
```diff
@@ -121,3 +121,13 @@ Finally, you should be able to see the introspected table structure and adjust t
...
+
+When using ClickHouse Cloud, you must first file a support case to enable the MySQL feature. Our support team will enable this feature, which then creates a dedicated user for the MySQL protocol.
+
+![Looker Studio Cloud Connection](./images/looker-studio-mysql-cloud.png)
+
+In the Looker Studio UI, choose the "Enable SSL" option. Upload the [LetsEncryptRoot Cert](https://letsencrypt.org/certificates/) as the server cert. ClickHouse Cloud's SSL certificate is signed by LetsEncrypt. You can download this root cert [here](https://letsencrypt.org/certs/isrgrootx1.pem).
```
Comment (slvrtrn) [inline docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md:131]: Do we need the first LetsEncryptRoot Cert link? We could just provide a link to download the pem.

File: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md (hunk 2 above)
Comment (slvrtrn) [reply docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md:125]: Maybe even above that?

File: docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md (hunk 3 above)
Comment (incfly) [reply docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md:131]: Let me reword. We can link the first link for the signed by LetsEncrypt. The first link is to comfort the users who are very sensitive about security. To give context why this cert, etc.

Comment (incfly) [reply docs/en/integrations/data-visualization/looker-studio-and-clickhouse.md]: updated.

**Pull request merged**